source/test/lysis_mode_test/Phage.test.cc

```diff
@@ -662,6 +662,41 @@ TEST_CASE("Phage ProcessResources", "[lysis]"){
             bacterium.Delete();
         }
     }
+
+    THEN("Phage, not Symbiont, ProcessResources is used"){
+      double int_val = 0.2;
+
+      config.LYSIS(1);
+      config.LYSIS_CHANCE(0);
+      config.BENEFIT_TO_HOST(0);
+      config.SYNERGY(5);
+
+      emp::Ptr<Organism> phage = emp::NewPtr<Phage>(random, world, &config, int_val);
+      emp::Ptr<Host> bacterium = emp::NewPtr<Bacterium>(random, world, &config, int_val);
+      phage->UponInjection();
+
+      emp::Ptr<Organism> symbiont = emp::NewPtr<Symbiont>(random, world, &config, int_val);

```

Comment:
I assume you had these to check what the symbiont's processresources would do, but they shouldn't be in this test. You could put in a comment about what the symbiont's process resources would result in and versus the phage's process resources, but the test should be just making the phage and bacterium and calling DistribResToSym and making sure that the amount the results is correct.